cxx/distributions/beta_bernoulli.cc

```diff
@@ -49,7 +49,11 @@ void BetaBernoulli::transition_hyperparameters(std::mt19937* prng) {
       }
     }
   }
-  int i = sample_from_logps(logps, prng);
-  alpha = hypers[i].first;
-  beta = hypers[i].second;
+  if (logps.empty()) {
+    printf("Warning! All hyperparameters for BetaBernoulli give nans!\n");
```

Comment:
I guess practically inference gets stuck, because the hyperparameters don't move, but there might be some hope for other parameters to move and get unstuck? My thinking is these should be asserts since it would be hard to get out of here (you need the observations to change, so the cluster assignments to change) and highlights some suboptimality of inference that should be fixed somewhere else (either numerical stability, bad preconditioning of some sort, etc).

Reply:
Done.